src/AspNetCoreModuleV2/RequestHandler/outofprocess/winhttphelper.cxx

```diff
@@ -26,11 +26,6 @@ WINHTTP_HELPER::StaticInitialize(
 {
     HRESULT hr = S_OK;
 
-    if (!g_fWebSocketSupported)
```

Comment:
We were shortcitcuiting here, is removing it fine?

Reply:
Reverting.